src/snowflake/cli/_plugins/connection/util.py

```diff
@@ -76,26 +72,22 @@ def get_ui_parameter(
 
 
 @lru_cache()
-def get_ui_parameters(conn: SnowflakeConnection) -> Dict[UIParameter, str]:
+def get_ui_parameters(conn: SnowflakeConnection) -> Dict[UIParameter, Any]:
     """
     Returns the UI parameters from the SYSTEM$BOOTSTRAP_DATA_REQUEST function
     """
 
-    parameters_to_fetch = sorted([param.value for param in UIParameter])
+    parameters_to_fetch = [param.value for param in UIParameter]
 
-    query = dedent(
-        f"""
-        select value['value']::string as PARAM_VALUE, value['name']::string as PARAM_NAME from table(flatten(
-            input => parse_json(SYSTEM$BOOTSTRAP_DATA_REQUEST()),
-            path => 'clientParamsInfo'
-        )) where value['name'] in ('{"', '".join(parameters_to_fetch)}');
-        """
-    )
+    query = "call system$bootstrap_data_request('CLIENT_PARAMS_INFO')"
```

Comment:
I'm afraid the knowledge will get lost quickly here, please add a comment regarding how this logic avoids requiring a warehouse.

Reply:
added comment.